crm_phonecall/views/crm_phonecall_view.xml

```diff
@@ -100,6 +100,11 @@
                         <field name="priority" widget="priority"/>
                         <field name="opportunity_id" context="{'opportunity_id': opportunity_id}"/>
                     </group>
+                    <group string="Tracking" groups="base.group_multi_company,base.group_no_one" name="categorization">
```

Comment:
Why multi-company here?

Reply:
I copied it from [the crm.lead view](https://github.com/odoo/odoo/blob/c14eb9ae9e29977be27bc0370d7662a6acd6ab9a/addons/crm/crm_lead_view.xml#L181), thinking that it would be confusing for a user to see the fields in crm.phonecall and not in crm.lead.

Reply:
That one includes the field `company_id`, so that's the reason. That's a bug clearly as you can use tracking info without multi-company. Please remove it here and optionally make the PR in Odoo

Reply:
Then do I leave the `base.group_no_one` in place?

Reply:
Yes, you can leave that one considering these fields as "technical".